agent/command/git_merge_pr.go

```diff
@@ -72,6 +73,14 @@ func (c *gitMergePR) Execute(ctx context.Context, comm client.Communicator, logg
 	if token == "" {
 		token = conf.Expansions.Get(evergreen.GlobalGitHubTokenExpansion)
 	}
+	appToken := conf.Expansions.Get(evergreen.GithubAppToken)
+	if appToken == "" {
+		logger.Task().Debug(message.Fields{
```

Comment:
this doesn't log to splunk since it's in the agent so kind of noisy for users

Reply:
removed